### PR TITLE
Use the core shim in the main test case

### PR DESCRIPTION
**Pull request by srtfisher**

### CODE REVIEW

**Review by anubisthejackle — APPROVED** [review]
:guitar:

File: src/mantle/testing/concerns/trait-wordpress-state.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ trait WordPress_State {
 	/**
 	 * Cleans the global scope (e.g `$_GET` and `$_POST`).
 	 */
-	public static function clean_up_global_scope() {
+	public function clean_up_global_scope() {
```
Comment (renatonascalves) [inline src/mantle/testing/concerns/trait-wordpress-state.php:21]: Should you change here to use `$this`?

https://github.com/alleyinteractive/mantle-framework/blob/4051a5f2891c44b6b5fc15a4f0b9b5fe1c8d9e8f/src/mantle/testing/class-test-case.php#L156

**Review by renatonascalves — APPROVED** [review]
🗡️

File: src/mantle/testing/concerns/trait-wordpress-state.php (hunk 1 above)
Comment (anubisthejackle) [reply src/mantle/testing/concerns/trait-wordpress-state.php:21]: +1 to this

Comment (srtfisher) [reply src/mantle/testing/concerns/trait-wordpress-state.php:21]: We all know this makes better sense as a static method 😭

**Pull request merged**